templates/aws/policies/trust-policy.json

```diff
@@ -18,7 +18,7 @@
         {
             "Effect": "Allow",
             "Principal": {
-                "AWS": "arn:aws:iam::{{.accountID}}:role/eodhp-dev-workspace-services-role"
+                "AWS": "arn:aws:iam::{{.accountID}}:role/WorkspaceServices-{{.cluster}}"
```

Comment:
I would just pass the the whole role name in so we're not hardcoding any part of it. Saves tying us to a particular naming convention.

Reply:
Done